### PR TITLE
use buoy exists

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

**Comment by n3wscott** [comment]
/hold

Testing.

File: .github/workflows/knative-releasability.yaml [hunk 1]
```diff
@@ -19,35 +19,63 @@ name: 'Releasability'
...
+        id: exists
+        run: |
+          EXISTS=0
+          buoy exists go.mod --release ${RELEASE} --verbose || EXISTS=$?
+          if [[ "$EXISTS" -eq "0" ]]; then
+            EXISTS=true
+          else
+            EXISTS=false
+          fi
+          echo ::set-output name=release-branch::${EXISTS}
```
Comment (mattmoor) [inline .github/workflows/knative-releasability.yaml:75]: I believe this needs to be an env var

Comment (mattmoor) [reply .github/workflows/knative-releasability.yaml:75]: outputs set this way only work across jobs IIUC

Comment (n3wscott) [reply .github/workflows/knative-releasability.yaml:75]: I found other examples of this in the docs.

```
      - name: Set selected color
        run: echo '::set-output name=SELECTED_COLOR::green'
        id: random-color-generator
      - name: Get color
        run: echo "The selected color is ${{ steps.random-color-generator.outputs.SELECTED_COLOR }}"
```

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions

Comment (mattmoor) [reply .github/workflows/knative-releasability.yaml:75]: well then I wonder what I screwed up (or if github quietly fixed it) 🤔

**Review by mattmoor — COMMENTED** [review]
/lgtm

**Pull request merged**